pkg/permissions/generate.go

```diff
@@ -19,6 +20,80 @@ type AgentRBACManifests struct {
 	RoleBindings []rbac.RoleBinding
 }
 
+//*******************************
+func AgentCLR(clusterRoles []rbac.ClusterRole) string {
+	var got []string
+	for _, clr := range clusterRoles {
+		var apiGroups, resources, verbs string
+		for _, rule := range clr.Rules {
+			apiGroups = strings.Join(rule.APIGroups, "\", \"")
+			resources = strings.Join(rule.Resources, "\", \"")
+			verbs = strings.Join(rule.Verbs, "\", \"")
+		}
+
+		got = append(got, fmt.Sprintf(
+			`apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+	name: jetstack-secure-agent-pods-reader
+rules:
+- apiGroups: ["%s"]
+	resources: ["%s"]
+	verbs: ["%s"]`, apiGroups, resources, verbs))
+	}
+	out := strings.Join(got, "\n")
+	return out
+}
+
+//*******************************
+func AgentCLRB(clusterRoleBindings []rbac.ClusterRoleBinding) string {
+	var got []string
+	for _, clrb := range clusterRoleBindings {
+		got = append(got, fmt.Sprintf(
+			`---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: %s
+roleRef:
+  kind: ClusterRole
+  name: %s
+  apiGroup: %s
+subjects:
+- kind: %s
+  name: %s
+  namespace: %s`, clrb.ObjectMeta.Name, clrb.RoleRef.Name, clrb.RoleRef.APIGroup, clrb.Subjects[0].Kind, clrb.Subjects[0].Name, clrb.Subjects[0].Namespace))
+	}
+	out := strings.Join(got, "\n")
+	return out
+}
+
+//*******************************
+func AgentRB(RoleBindings []rbac.RoleBinding) string {
```

Comment:
```suggestion
func AgentRB(roleBindings []rbac.RoleBinding) string {
```

Function parameters should be camel-cased